### ISSUE TITLE
Installing with npm still gives error reported in #213

### ISSUE THREAD

**Issue by ortegag** [peterbraden/node-opencv#222]
I'm trying to install with npm and I'm still getting the issue described in #213 and supposedly fixed by #218. I've tried cleaning npm cache but with no luck. Is it missing publication or something?

**Comment by amchang** [comment]
It's not, you have to npm install peterbraden/node-opencv at the moment....

**Comment by peterbraden** [comment]
Yep, I need to update the version, please `npm install peterbraden/node-opencv` for now - thanks!

**Comment by alamcordeiro** [comment]
Hello i am get this error:
 
npm ERR! opencv@3.0.0 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1

MAC OS X 10.10.2
OpenCV 3.0
NPM 2.5.1

Running
```shell
 # npm install opencv
 
```
OR 
```shell
# npm install peterbraden/node-opencv
```

npm-debug.log: https://gist.github.com/alamcordeiro/dc61ff1fe09a958ca83c

**Comment by idofishler** [comment]
I did try the above `npm install peterbraden/node-opencv` and still got the same issue...
